src/domain/collaboration/callouts-set/callouts.set.service.ts

```
@@ -465,8 +459,14 @@ export class CalloutsSetService {
       const hasAccess = this.hasAgentAccessToCallout(callout, agentInfo);
       if (!hasAccess) return false;
 
-      // Filter by Callout types
-      if (args.types && !args.types.includes(callout.type)) {
+      // Filter by Contribution types
+      if (
+        args.withContributionTypes &&
+        args.withContributionTypes.length &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)